pkg/result/destination.go

```diff
@@ -0,0 +1,24 @@
+package result
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func CreateDestination(destination string) (*os.File, error) {
+	if !strings.Contains(destination, ".") {
+		destination += ".txt"
+	}
+	return os.Create(filepath.Join(destination))
+}
+
+func Write(file *os.File, elem ...string) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function Write should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)